internal/config/node/cgroups_linux.go

```diff
@@ -26,6 +27,38 @@ var (
 	cgroupIsV2Err error
 )
 
+// CgroupHierarchy is a cgroup version agnostic struct that allows
```

Comment:
I don't think this is best here. This package has a pretty particular use (for initializing variables that CRI-O needs to react to on the node). If it's not being used anywhere else, I'd prefer it in runtime_handler_hooks